examples/basic/main.tf

```diff
@@ -53,12 +27,16 @@ module "test_recipe" {
 }
 
 module "test_pipeline" {
-  source  = "rhythmictech/imagebuilder-pipeline/aws"
-  version = "~> 0.3.0"
+  source = "rhythmictech/imagebuilder-pipeline/aws"
```

Comment:
are these unversioned?

Reply:
It's an example. I feel like version pinning in examples is not our/the norm